Oleksii_Fedorchuk/HW5_Oleksii_Fedorchuk/task2.py

```diff
@@ -1,10 +1,19 @@
 friends = ["John", "Marta", "James", "Amanda", "Marianna"]
 title_string = ' Names '
 print(f"{title_string}".center(50, "*"))
-for names in friends:
-    print(f"{friends[0]}".rjust(27))
-    print(f"{friends[1]}".rjust(27))
-    print(f"{friends[2]}".rjust(27))
-    print(f"{friends[3]}".rjust(27))
-    print(f"{friends[4]}".rjust(27))
-    break
+print(f"{friends[0].rjust(27)}")
+print(f"{friends[1].rjust(27)}")
+print(f"{friends[2].rjust(27)}")
+print(f"{friends[3].rjust(27)}")
+print(f"{friends[4].rjust(27)}")
+
```

Comment:
Why you do not use just:
for friend in friends:
       print(friend.rjust(27))

Reply:
Yes, you are right. It's  much easier.